packages/reacord/scripts/discordjs-manual-test.tsx

```diff
@@ -50,7 +50,7 @@ const createTest = async (
 }
 
 await createTest("basic", (channel) => {
-	reacord.send(channel.id, "Hello, world!")
+	reacord.createChannelMessage(channel, {}, "Hello, world!")
```

Comment:
This would require us to use a react node for the instance initialization usage i.e. to create an instance with proposed API we can use the following syntax:

```jsx
const instance = reacord.createChannelMessage(channel, { someOption: "e" })
```

Otherwise to provide options we'd need to provide additional argument:

```jsx
const instance = reacord.createChannelMessage(
  channel,
  <></>, // Or something else from ReactNode type
  { someOption: "e" }
)
```

Alternatively we can add additional validation step for 2nd argument e.g. to provide options & content we use following options API:

```jsx
const instance = reacord.createChannelMessage(
  channel,
  {
    content: <HelloWorld />, // or children/node ¯\_(ツ)_/¯
    someOption: "e"
  }
)
// Or simply
const instance = reacord.createChannelMessage(channel, <HelloWorld />)
```

Both ways are good but I prefer avoiding redundant initial content usage.

Reply:
While that's true, I _feel_ like it's more common to have initial content without options, than to have options without content. Although I'm not confident on that, since lots of command replies tend to be ephemeral...

Alternatively, we could remove the content argument together and rely only on the render method:
```ts
// render now
reacord.createChannelMessage(channel).render(content)
reacord.createChannelMessage(channel, options).render(content)

// render later
const instance = reacord.createChannelMessage(channel)
const instance = reacord.createChannelMessage(channel, options)

// ...

instance.render(content)
```

This also mirrors React's `createRoot` API, so I'd feel pretty comfortable with it 🙂

Reply:
Mirroring react's API sounds like a good solution & compromise

Reply:
Reacord's create methods now follow the `.render()` only API. Just updated guides.